RELEASE.md

```diff
@@ -13,24 +13,29 @@ The Tag identifier format is:
 ## Step 1: Prepare an appropriate log message for the release
 
 Take a look at the changes from the last release to now within the project
-subdirectory you're going to release from.
+subdirectory you're going to release from as well as `internal` if the project
+relies on code living there.
 
-`git log v2.0.0..HEAD -p proxy-init`
+`git log proxy-init/v2.1.0..HEAD -p proxy-init`
+`git log proxy-init/v2.1.0..HEAD -p internal`
 
 Prepare a commit message. For each meaningful commit since the last tag, add
 a single line. Use `git show` on another release tag to see the format we use.
 
+Note: currently `validator` doesn't rely on `internal`, while `cni-plugin` and
+`proxy-init` do.
+
 ## Step 2: Tag the release
 
-`git tag -a proxy-init/v2.1.0`
+`git tag -a proxy-init/v2.2.0`
```

Comment:
nit: we could also post a quick command here on how we can get the latest tag for a particular subproject:

e.g

```
## Step 2: Tag the release

Find the current latest version for the subproject

`git tag -l '<proxy-init | cni-plugin | validator>/*' --sort=-"v:refname" | head -n 1`

Increase the version with a major, minor, or patch according to the changes made

`git tag -a proxy-init/v2.x.x`

```

Don't know though, it might complicate everything.